temporal-serviceclient/src/main/java/io/temporal/serviceclient/CloudServiceStubsOptions.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2022 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ * Copyright (C) 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this material except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.temporal.serviceclient;
+
+import io.grpc.ManagedChannel;
+import java.util.Objects;
+
+/**
+ * Options for cloud service.
+ *
+ * <p>WARNING: The cloud service is currently experimental.
+ */
+public final class CloudServiceStubsOptions extends ServiceStubsOptions {
```

Comment:
Could we add a `toString`? users have complained in the past that *Options didn't implement `toString`

Reply:
I specifically excluded this because it's not in `OperatorServiceStubsOptions` and I'd argue it was improperly implemented on `WorkflowServiceStubsOptions` because it doesn't even include all the settings from the inherited class. But if I do need to make it just a string with the one `version` option I can.

Reply:
If you think the `WorkflowServiceStubsOptions ` is wrong can you open an issue? I am fine to defer adding a `toString` till that issue since then there will be a lot more options included.

Reply:
Opened https://github.com/temporalio/sdk-java/issues/2148